src/BackgroundLayerChooser/BackgroundLayerChooser.tsx

```diff
@@ -132,10 +136,19 @@ export const BackgroundLayerChooser: React.FC<BackgroundLayerChooserProps> = ({
                   backgroundLayerFilter={backgroundLayerFilter}
                   zoom={zoom}
                   center={center}
+
                 />
               );
             })
           }
+          { allowEmptyBackground &&
+          <ToggleButton
+            onToggle={() => { selectedLayer?.setVisible(false); }}
```

Comment:
Haven't checked in detail, but this _might_ result in an unwanted behaviour if the selected layer is not available. Probably it's safer to just loop over all `layers` and set their `visibility` to `false`.

But again, I'm not sure if this is needed.